modules/test/playwright/tests/commerce/commerce-currency-web/commerceAdminCurrency.spec.ts

```diff
@@ -0,0 +1,223 @@
+	await commerceAdminCurrenciesPage.search.fill(currencyName);
+	await commerceAdminCurrenciesPage.searchButton.click();
+
+	await expect(
+		commerceAdminCurrenciesPage.currencyNameLink(currencyName)
+	).toBeVisible();
+
+	await commerceAdminCurrenciesPage.actionsButton.click();
+	await commerceAdminCurrenciesPage.activeToggleMenuItem.click();
+
+	await waitForAlert(page);
+
+	await commerceAdminCurrenciesPage.search.click();
+	await commerceAdminCurrenciesPage.search.fill(currencyName);
+	await commerceAdminCurrenciesPage.searchButton.click();
+	await commerceAdminCurrenciesPage.actionsButton.click();
+	await commerceAdminCurrenciesPage.primaryMenuItem.click();
+
+	await waitForAlert(page);
+
+	await commerceAdminCurrenciesPage.currencyNameLink(currencyName).click();
+
+	await expect(commerceAdminCurrencyDetailsPage.activeToggle).toBeChecked();
+	await expect(commerceAdminCurrencyDetailsPage.primaryToggle).toBeChecked();
+
+	await commerceAdminCurrencyDetailsPage.saveButton.click();
+
+	await waitForAlert(page);
+
+	await commerceAdminCurrenciesPage.currencyNameLink('US Dollar').click();
+
+	await expect(
+		commerceAdminCurrencyDetailsPage.primaryToggle
+	).not.toBeChecked();
+
+	await commerceAdminCurrencyDetailsPage.primaryToggle.setChecked(true);
+	await commerceAdminCurrencyDetailsPage.saveButton.click();
+
+	await waitForAlert(page);
+
+	await commerceAdminCurrenciesPage.search.click();
+	await commerceAdminCurrenciesPage.search.fill(currencyName);
+	await commerceAdminCurrenciesPage.searchButton.click();
+	await commerceAdminCurrenciesPage.actionsButton.click();
+
+	let alertText = '';
+	page.on('dialog', (dialog) => {
+		alertText = dialog.message();
+		dialog.accept();
+	});
+
+	await commerceAdminCurrenciesPage.deleteMenuItem.click();
+
+	await expect(alertText).toEqual(
+		'Are you sure you want to delete this entry?'
+	);
+
+	await waitForAlert(page);
+
+	await expect(
+		commerceAdminCurrenciesPage.currencyNameLink(currencyName)
+	).not.toBeVisible();
+});
+
+test('COMMERCE-9936 A disabled default currency should not be usable', async ({
+	accountsPage,
+	apiHelpers,
+	applicationsMenuPage,
+	commerceChannelDefaultsPage,
+	editAccountPage,
+	page,
+}) => {
+	test.setTimeout(180000);
+	try {
+		const {site} = await miniumSetUp(apiHelpers);
+
+		const account = await apiHelpers.headlessAdminUser.postAccount({
+			name: getRandomString(),
+			type: 'business',
+		});
+
+		apiHelpers.data.push({id: account.id, type: 'account'});
+
+		await accountsPage.goto();
+		await (await accountsPage.accountsTableRowLink(account.name)).click();
+		await editAccountPage.channelDefaultsLink.click();
+
+		await commerceChannelDefaultsPage.defaultCommerceCurrenciesButton.click();
+		await commerceChannelDefaultsPage.editFrameCurrencySelect.selectOption(
+			'Euro'
+		);
+
+		await commerceChannelDefaultsPage.editFrameSaveButton.click();
+
+		const currencies =
+			await apiHelpers.headlessCommerceAdminCatalog.getCurrenciesPage(
+				'EUR'
+			);
```

Comment:
can we move this before the try? So in the finally we don't have to do the get again